odbp.client/src/components/GppWooInfoPopover.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <slot v-if="isPopoverSupported" name="trigger" :trigger-props="triggerProps">
+    <button type="button" v-bind="triggerProps">?</button>
+  </slot>
+
+  <div
+    ref="tooltipRef"
+    popover
+    role="tooltip"
+    :id="tooltipId"
+    class="utrecht-tooltip gpp-woo-info-popover"
+    @click.prevent
+  >
+    <utrecht-icon icon="info" />
+
+    <slot></slot>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { computed, ref, useId } from "vue";
+import { onKeyStroke, useSupported } from "@vueuse/core";
+import UtrechtIcon from "@/components/UtrechtIcon.vue";
+
+const tooltipId = useId();
+
+const tooltipRef = ref<HTMLElement>();
+
+const triggerProps = computed(() => ({
+  popovertarget: tooltipId,
+  "aria-describedby": tooltipId
+}));
+
+onKeyStroke("Tab", () => tooltipRef.value?.hidePopover());
```

Comment:
Voor de zekerheid, voor in browsers zonder popover support?
```suggestion
onKeyStroke("Tab", () => tooltipRef.value?.hidePopover?.());
```